emcc.py

```diff
@@ -2404,7 +2396,12 @@ def check_bad_eq(arg):
       requested_level = newargs[i][2:] or '3'
       options.debug_level = validate_arg_level(requested_level, 4, 'Invalid debug level: ' + newargs[i])
       options.requested_debug = newargs[i]
-      newargs[i] = ''
+      if options.debug_level >= 3:
+        # make sure debug info is preserved (note: clang won't accept -g4)
+        newargs[i] = '-g'
+      else:
+        # don't pass -g1 or -g2 to clang
+        newargs[i] = ''
```

Comment:
Why not?

Reply:
I'm not sure what it does in clang, but we've never passed it, so I'm not sure it's worth the risk of a change here.